src/app/status/node-status.tsx

```diff
@@ -1,10 +1,18 @@
 'use client'
 
 import { NodeStatusPill } from '@/components/status/node-status-pill'
-import { NodeStatusSchema } from '@/data/requests'
+import { getNodeData, NodeStatusSchema } from '@/data/requests'
 import { useNodeData } from '@/lib/hooks/use-node-data'
+import { useEffect, useState } from 'react'
 
-export const NodeStatus = ({ initialData }: { initialData?: NodeStatusSchema }) => {
+export const NodeStatus = () => {
+  const [initialData, setInitialData] = useState<NodeStatusSchema>()
+  useEffect(() => {
+    ;(async () => {
+      const data = await getNodeData().catch(() => undefined)
+      setInitialData(data)
+    })()
+  }, [])
```

Comment:
the initial data was meant to get data from the server side, so the initial render will show some data.